### PR TITLE
feat: add maxValue and minValue prop to AreaChart, BarChart, and LineChart

### PR DESCRIPTION
**Pull request by sam-b-rose**
Adds an optional `maxValue` and `minValue` prop to the following components: `AreaChart`, `BarChart`, and `LineChart`.

This allows for more flexibility setting the [`domain`](https://recharts.org/en-US/api/YAxis#domain) of the `YAxis` component from Recharts.

This is needed for a use-case were the domain might be a percentage (0-100) and the data is only a small percentage.

Cheers!

storybook-example

### CODE REVIEW

**Comment by sam-b-rose** [comment]
Hi @mitrotasios or @christopherkindl! 👋 sorry to bother you directly, but any thoughts on this enhancement? Something we could consider adding?

**Comment by mitrotasios** [comment]
Hey @samrose3 sorry for the delayed response. Yeah I think that definitely makes sense, especially when you have percentage values as you mentioned. To keep it consistent I would suggest something like `minValue` and `maxValue`. Wdyt? We could include this in our next minor release potentially.

**Comment by sam-b-rose** [comment]
@mitrotasios perfect, I like `minValue` and `maxValue` 👍  I've updated the PR and added some stories to showcase/test the props.

**Comment by mitrotasios** [comment]
awesome thanks! How urgent is this for you?

**Comment by sam-b-rose** [comment]
Of course, happy to contribute! Awesome library.

> How urgent is this for you?

Moderate urgency. We'd like to have the feature sooner than later to help complete a product feature, but not critical 👍

**Comment by mitrotasios** [comment]
cool, we'll try to merge this as soon as we can. Would you maybe like to hop on a call at any time? We would love to hear feedback and more about your use case in general. Feel free to email me: achi@tremor.so

**Comment by sam-b-rose** [comment]
Hi @mitrotasios just checking in on this. What is the Tremor release cadence? And anything I can help with that may be blocking this from merging into `main`?

**Comment by mitrotasios** [comment]
Hey @samrose3. There is no regular release cadence unfortunately because we work full time and our schedule changes here and there. We will release this most likely on Monday the latest though.

**Comment by mitrotasios** [comment]
hey @samrose3, I made a few changes to allow only numeric values as inputs for the `minValue` and `maxValue` props. We'd like to keep the API framework agnostic and avoid using recharts-specific inputs as we are considering migrating to another chart library eventually. I hope that's ok. Will merge this to beta now. Thanks a lot for the work! 🤗

**Comment by mitrotasios** [comment]
@samrose3 JFYI, we will merge this to main tomorrow probably, but you can already use the beta version by npm installing `@tremor/react@beta-feat-1.3.0` if you like :)

**Comment by sam-b-rose** [comment]
Woohoo!! Happy to help and thank you for accepting the contribution 🚀

File: src/components/chart-elements/common/utils.ts [hunk 1]
```diff
@@ -10,3 +10,14 @@ export const constructCategoryColors = (
...
 };
+
+export const getYAxisDomain = (
+    autoMinValue: boolean,
+    minValue: number | undefined,
+    maxValue: number | undefined
+) => {
+    const minDomain = autoMinValue ? 'auto' : (minValue ?? 0);
+    const maxDomain = maxValue ?? 'auto';
+    console.log([minDomain, maxDomain]);
```
Comment (sam-b-rose) [inline src/components/chart-elements/common/utils.ts:21]: Prop is working great but I noticed this sneaky console.log while testing out the `beta` release. Hope this helps https://github.com/tremorlabs/tremor/pull/244

Comment (mitrotasios) [reply src/components/chart-elements/common/utils.ts:21]: dang good catch, thanks a lot 🙏  We were about to release it :D

**Pull request merged**